packages/channel-client/tests/index.test.ts

```diff
@@ -0,0 +1,15 @@
+import ChannelClient from '../src';
+import {IChannelProvider} from '@statechannels/channel-provider';
+
+// TODO: Figure out how to test
+describe('ChannelClient', () => {
+  it('instantiates', () => {
+    expect(
+      new ChannelClient({
+        enable: () => {
+          /* do nothing */
+        }
+      } as IChannelProvider)
+    ).toBeDefined();
```

Comment:
Not directly relevant to the scope at hand but we should probably stick to the same interface naming convention (`IChannelProvider` here vs `ChannelClientInterface` elsewhere)